plans/example-browser/README.md

```diff
@@ -0,0 +1,35 @@
+# Example Browser Testplan
```

Comment:
Is this plan destined to be removed and replaced by the `example-browser-node`?

Reply:
I was under the impression that this was not the case, but we can remove it if you want it.
To me it is fine to stay as this is a very different approach. In the original plan we run the test from within NodeJS, and do not try to run testground-sdk in the browser.

Reply:
@laurentsenta want me to remove it, or what is your take on it?

Reply:
If I understand correctly:
they both answer the same question: "as a test author, how do I create a test that runs in a browser runtime?" and there's no obvious reason to use the version that can't access the testground sdk.

I would like to make sure we have one "obvious" way to do things, that we might make "Testground native" eventually. It will be easier for maintainers to improve and communicate about it, and it'll be easier for users to get started and rely on our code.

Reply:
Fine for now, it's anyway in the Git history so we can always refer to the version on the older commit if we ever have to. Will delete it. RIP.